decompressed/base/etc/init.d/rootdevice

```diff
@@ -122,7 +124,6 @@ setup_network() {
 	if [ ! "$(uci -q get network.lan.pppoerelay)" ]; then
 		uci -q add_list network.lan.pppoerelay=''
 	fi
-	sed -i -e 's/option pppoerelay/list pppoerelay/g' /etc/config/network
```

Comment:
Whaat???? I didn't delete this?!

Reply:
probably you started editing before https://github.com/Ansuel/tch-nginx-gui/commit/a67c252ff098218529fa5cb8e0b654ea625013bc

Reply:
Ok